### DISCUSSION TITLE
I cannot sign-in to the dashboard.

### DISCUSSION THREAD

**Discussion by panicstyle** [bcnmy/biconomy-client-sdk#464] [General]
I cannot sign-in to the dashboard.

After entering the e-mail address, I connected to the link in the received email and clicked Verify, but the following error occurred. "Authentication error Oops! Something went wrong during the sign in process. Please try signing in again."

The same thing happens if you select “Sign in with github”.

Is there something I need to do?

**Comment by himanshugarg06** [comment]
I hope its working now. Its being fixed.
